FDBLibTLS/FDBLibTLSVerify.cpp

```diff
@@ -136,7 +136,7 @@ static std::pair<std::string, std::string> splitPair(std::string const& input, c
 static int abbrevToNID(std::string const& sn) {
 	int nid = NID_undef;
 
-	if (sn == "C" || sn == "CN" || sn == "L" || sn == "ST" || sn == "O" || sn == "OU" || sn == "UID" || sn == "DC")
+	if (sn == "C" || sn == "CN" || sn == "L" || sn == "ST" || sn == "O" || sn == "OU" || sn == "UID" || sn == "DC" || sn == "subjectAltName")
```

Comment:
To preemptively address a likely review comment, "subjectAltName" is indeed the official "short name" for Subject Alternative Name.  See [OpenSSL's listing of names](https://github.com/krzyzanowskim/OpenSSL/blob/ed6dc693f2fdacc37cd7eee3d58b017f3a56796a/include-ios/openssl/objects.h#L475).